### PR TITLE
Add support reg update packages

### PR DESCRIPTION
**Pull request by swadeley**
Hello

An update to support the "Update packages" check box on the global registration form, advanced tab.

Required for:
[ Add test update packages checkbox #9380 ](https://github.com/SatelliteQE/robottelo/pull/9380)

Thank you

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
@swadeley Can you please rebase your changes?

**Comment by swadeley** [comment]
Rebased

**Review by omkarkhatavkar — APPROVED** [review]

File: airgun/entities/host.py [hunk 1]
```diff
@@ -253,7 +253,9 @@ def get_webconsole_content(self, entity_name):
         view.validations.assert_no_errors()
         # switch to the last opened tab,
         self.browser.switch_to_window(self.browser.window_handles[-1])
-        self.browser.wait_for_element(locator='//div[@id="index-brand"]', exception=True)
+        self.browser.wait_for_element(
+            locator='//div[@id="index-brand"]', exception=True, timeout=10
```
Comment (jyejare) [inline airgun/entities/host.py:257]: I believe the timeout is very less, is this expected and intentional ?

**Review by jyejare — APPROVED** [review]
ACK with a doubtful question.

File: airgun/entities/host.py (hunk 1 above)
Comment (swadeley) [reply airgun/entities/host.py:257]: Hi, I can't see how this is related to my change

**Comment by swadeley** [comment]
Hi, this PR is also required some other tests in tests/foreman/ui/test_host.py

File: airgun/entities/host.py (hunk 1 above)
Comment (mshriver) [reply airgun/entities/host.py:257]: @swadeley your commits are off, looks like you might have rebase/amended too far back. That's why this is showing up in your PR.

Comment (swadeley) [reply airgun/entities/host.py:257]: I can rebase again?

Comment (swadeley) [reply airgun/entities/host.py]: @jyejare @mshriver I deleted that commit and mine, rebased, c-picked my commit, should be OK now. Now idea how it happened, honest.

**Comment by swadeley** [comment]
> Hi, this PR is also required some other tests in tests/foreman/ui/test_host.py

sorry, it is the companion robottelo PR thats blocking some tests in `tests/foreman/ui/test_host.py`

**Comment by swadeley** [comment]
Thank you

**Pull request merged**